earth2mip/initial_conditions/ifs.py

```diff
@@ -49,75 +50,109 @@ def _get_channel(c: str, **kwargs) -> xarray.DataArray:
         return kwargs[varcode].interp(isobaricInhPa=pressure_level)
 
 
-def get(time: datetime.datetime, channels: List[str]):
-    root = "https://ecmwf-forecasts.s3.eu-central-1.amazonaws.com/"
-    path = root + _get_filename(time, "0h")
-    local_path = filesystem._download_cached(path)
-    dataset_0h = xarray.open_dataset(local_path, engine="cfgrib")
-
-    # get t2m and other things from 12 hour forecast initialized 12 hours before
-    # The HRES is only initialized every 12 hours
-    path = root + _get_filename(time - datetime.timedelta(hours=12), "12h")
-    local_path = filesystem._download_cached(path)
-    forecast_12h = xarray.open_dataset(local_path, engine="cfgrib")
-
-    channel_data = [
-        _get_channel(
-            c,
-            u10m=dataset_0h.u10,
-            v10m=dataset_0h.v10,
-            u100m=dataset_0h.u10,
-            v100m=dataset_0h.v10,
-            sp=dataset_0h.sp,
-            t2m=forecast_12h.t2m,
-            msl=forecast_12h.msl,
-            tcwv=forecast_12h.tciwv,
-            t=dataset_0h.t,
-            u=dataset_0h.u,
-            v=dataset_0h.v,
-            r=dataset_0h.r,
-            z=dataset_0h.gh * 9.81,
-        )
-        for c in channels
-    ]
+def get(time: datetime.datetime, channels: List[str], ensemble_member: int, 
+        root_path: str):
+    path = root_path + _get_filename(time, "0h")
```

Comment:
Does this assume that `root_path` ends in `'/'`? Could do `os.path.join(root_path, _get_filename(time, "0h"))`

Reply:
it did, updated as you suggested